doc/source/api/utilities/utilities.rst

```diff
@@ -131,28 +131,21 @@ PyVista Version Information
 The PyVista library provides a way of getting the version installed in your
 environment.
 
-.. code:: python
-
-   Output the version of PyVista.
-
-   >>> import pyvista
-   >>> pyvista.version_info
-   (0, 44, 0)
+>>> # Output the version of PyVista.
+>>> import pyvista
+>>> pyvista.version_info
+(0, 44, 0)
 
 VTK Version Information
 ~~~~~~~~~~~~~~~~~~~~~~~
 The PyVista library is heavily dependent on VTK and provides an easy
 way of getting the version of VTK in your environment.
 
-.. code:: python
-
-   Output the version of VTK.
-
-   >>> import pyvista
-   >>> pyvista.vtk_version_info
-   VTKVersionInfo(major=9, minor=1, micro=0)
-
-   Get the major version of VTK
+>>> # Output the version of VTK.
+>>> import pyvista
+>>> pyvista.vtk_version_info
+VTKVersionInfo(major=9, minor=1, micro=0)
```

Comment:
Was a mix of doctest and python code block. Need to see how it looks like in the documentation build

Reply:
Looks fine in the documentation artifacts (left is previous version, right is current).

![image](https://github.com/user-attachments/assets/89f2452a-f8d2-4f32-ace2-837ba46f0590)